CMakeLists.txt

```diff
@@ -167,9 +167,9 @@ message( STATUS "Software Version: ${VERSION}" )
 
 
 
-IF(${CMAKE_SYSTEM_NAME} MATCHES "Linux" OR ${CMAKE_SYSTEM_NAME} MATCHES "Darwin")
+IF(${CMAKE_SYSTEM_NAME} MATCHES "Linux" OR ${CMAKE_SYSTEM_NAME} MATCHES "Darwin" OR ${CMAKE_SYSTEM_NAME} MATCHES "Windows")
```

Comment:
I believe this covers all targets. I ser no harm now in including the CPackList for all. 

Please just remove the IF case and let it happen on all targets